tools/make_dictionary_file.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python2.7
+
+import sys
+"""
+Makes dictionary file from mozc
```

Comment:
Please write:
- The path to the output file you submitted
- Which file (file name) we should pass as an input of this script in https://github.com/google/mozc/tree/master/src/data/dictionary_oss